pkg/option/option_test.go

```diff
@@ -20,6 +21,10 @@ type optTestCase struct {
 
 // nolint: gocyclo
 func TestOption(t *testing.T) {
+	pcidb, err := pcidb.New()
```

Comment:
@OrbintSoft on Windows hosts, by default, this won't work because pcidb does not fetch the pciids file over the network by default. This is why the tests are failing on the Windows CI machines.

To get around this, you will need to wrap this like so:

```go
var pciTest *optTestCase
if runtime.GOOS == "linux" {
    pciTest = &optTestCase		{
			name: "pcidb",
			opts: []*option.Option{
				option.WithPCIDB(pcidb),
				option.WithChroot("/my/chroot/dir"),
			},
			merged: &option.Option{
				Chroot: stringPtr("/my/chroot/dir"),
				PCIDB:  pcidb,
			},
		},
    }
}
```